src/python/pants/task/task.py

```diff
@@ -144,9 +139,8 @@ def alternate_target_roots(cls, options, address_mapper, build_graph):
     """
 
   @classmethod
-  def _prepare(cls, options, round_manager):
+  def invoke_prepare(cls, options, round_manager):
     # Subclasses should not generally need to override this method.
```

Comment:
I initially wanted to remove the method, but then the caller (round_engine.py) would have to compute the task scope itself, and it's nice to have the awareness of options scoping encapsulated in the task class.